### PR TITLE
change version and remove unused values

### PR DESCRIPTION
**Pull request by haibinxie**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

File: charts/portworx/values.yaml [hunk 1]
```diff
@@ -3,9 +3,7 @@
 deployOperator: true                    # Deploy the Portworx operator
 deployCluster: true                     # Deploy the Portworx cluster
 
-deploymentType: oci                     # accepts "oci" or "docker"
-imageType: none                         #
-imageVersion: 2.9.1.4                   # Version of the PX Image.
+imageVersion: 2.11.0                    # Version of the PX Image.
```
Comment (piyush-nimbalkar) [inline charts/portworx/values.yaml:6]: We still default to 2.9.1.4 from spec generator, let's stick to that. 
https://install.portworx.com/version

File: charts/portworx/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: portworx
-version: 2.9.1
+version: 2.11.0
```
Comment (piyush-nimbalkar) [inline charts/portworx/Chart.yaml:2]: Let's update the version based on the px version. Check the other comment.

**Review by piyush-nimbalkar — APPROVED** [review]
Looks good after addressing the comments.

File: charts/portworx/Chart.yaml (hunk 2 above)
Comment (haibinxie) [reply charts/portworx/Chart.yaml]: Ok, I will keep it unchanged then, probably dont need the version suffix .4

**Comment by piyush-nimbalkar** [comment]
@haibinxie I believe we will have to update the index as well with the new artifact.

**Comment by piyush-nimbalkar** [comment]
Actually ignore my comment. Let's update the artifact when we merge the operator migration branch and bump up the chart version then.

**Comment by dsupure** [comment]
@piyush-nimbalkar , @haibinxie : yes, this is what we should do.
If you see the index, for the px, it's still using 2.9.1. We should bum up the chart version and starts to make operator helm installation as the default. 
https://github.com/portworx/helm/blob/master/stable/index.yaml

**Pull request merged**